public/components/trace_analytics/components/traces/services_list.tsx

```diff
@@ -58,14 +80,14 @@ export const ServicesList = ({
   );
 
   useEffect(() => {
+    // Update selectable options based on the current filtered service
     setOptions(
-      Object.keys(serviceMap).map((key) => {
-        return filteredService === key
-          ? { label: key, checked: 'on', bordered: false }
-          : { label: key, bordered: false };
```

Comment:
we don't need `bordered`?

Reply:
I think this setting is over-written and not needed. Checked with it set to true, false, and not included with no reflection in the UI.